src/main/java/converter/Conversion/StarterMetaDataConversion.java

```diff
@@ -91,10 +92,10 @@ public static void main(String[] args) throws Exception {
                 logger.debug("loading:'" + newFile.getAbsolutePath() + "' as RDF in ugh");
 
                 try {
-                    if (rdfInput.read(procFile.getAbsolutePath())) {
+                    rdfInput.read(procFile.getAbsolutePath());
```

Comment:
You are removing the if condition but previous condition block is now not correct indent (one level to deep)

Reply:
I found that the Eclipse function to corret indents seems not to work reliably. If I select the whole file (<kbd>Ctrl</kbd>+<kbd>A</kbd>) it works, if not, it does nothing. I don’t know if this is the desired behaviour, at least, it is confusing, as opposed to source code formatter. Was Fixed.